_includes/footer.html

```diff
@@ -46,6 +45,7 @@
             src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/brands/github.svg"
             alt="GitHub"
             class="brand-icon"
+	    aria-label="Github"
```

Comment:
This is already covered by [the `alt` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#Attributes).

```suggestion
```